README.md

```diff
@@ -75,8 +75,10 @@ Optional integration tests use Boost.Multiprecision and Boost.SIMD.
 2. Configure the project for development
 
    ```shell
-   cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_PROJECT_cnl_INCLUDE:FILEPATH="$(pwd)"/conan_paths.cmake ..
+   cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_PROJECT_cnl_INCLUDE:FILEPATH="$(pwd)"/conan_paths.cmake \ 
```

Comment:
Was this needed in order to enable tests, or was there some other problem? (I'd like to allow basic support without tool chain files so that the library isn't exclusive to a subset.)